### ISSUE TITLE
Austria City of Vienna - Open Government Data Adress API

### ISSUE THREAD

**Issue by sindre** [openaddresses/openaddresses#957]
Is it possible to add an web API or just full adress downloads?
The city of Vienna has an adress API under CC-BY 3.0 AT (https://creativecommons.org/licenses/by/3.0/at/deed.de)
Metadata (german): https://www.data.gv.at/katalog/dataset/c223b93a-2634-4f06-ac73-8709b9e16888
Dokumentation (german): http://www.data.wien.gv.at/pdf/adressservice-doku.pdf
Endpoint Exmaple: http://data.wien.gv.at/daten/OGDAddressService.svc/GetAddressInfo?Address=Geusaugasse%2047
Happy to help with the integration.

**Comment by okilimu** [comment]
in this github project, only address datasets or metadata about complete datasets will be stored

**Comment by sindre** [comment]
OK. Thanks. Should i close the issue?

**Comment by geobrando** [comment]
Hi @sindre .  Currently we're able to accept downloads or links to Esri Map- or Featureservers that contain address endpoints. But thank you for the tip.

**Comment by sbma44** [comment]
Hmm -- we have done work in the past that involves preprocessing/scrapes that go beyond the limits of what conform objects can do. Consider the contents of https://github.com/openaddresses/openaddresses/tree/master/scripts

If this API could be completely scraped and stored in a CSV, a lightweight conform object would be written to pull it in. Unfortunately I don't see any means of ensuring that a scrape is complete at the moment.

**Comment by okilimu** [comment]
The api in this case only gives you coordinates for a already known address, consisting of a street and housenumber. It does not allow to ask for alle housenumbers in a street, for example.

**Comment by geobrando** [comment]
@okilimu Yeah, if it's only a forward and not a reverse-geocoder there's nothing we can do with this.  Tried to take a look at the site but my German is non-existent.

**Comment by sindre** [comment]
Sorry, there is now complete dataset available. Only the API. Will inform if something changes.

**Comment by okilimu** [comment]
@sindre: do you mean "no complete" or "now a complete" dataset?
If you found the dataset, please give the link to the dataset, i didn't find it a few minutes ago.

**Comment by sindre** [comment]
sorry. it should mean "no complete".
